src/services/countries.js

```
@@ -0,0 +1,19 @@
+const axios = require('axios');
+
+const API_KEY = 'OWRxZlBwaXpIWXlYN3loZlB6cm1hdkhpWHVNeUthR1lFVlU3S21vWA==';
```

ESLint

[error] Extra semicolon.

(semi)